backend/src/main/java/de/bund/digitalservice/ris/norms/domain/entity/FRBR.java

```diff
@@ -0,0 +1,76 @@
+package de.bund.digitalservice.ris.norms.domain.entity;
+
+import de.bund.digitalservice.ris.norms.utils.NodeParser;
+import de.bund.digitalservice.ris.norms.utils.exceptions.MandatoryNodeNotFound;
+import java.time.LocalDate;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+import org.w3c.dom.NamedNodeMap;
+import org.w3c.dom.Node;
+
+/**
+ * Class representing a generalization of FRBR elements. It cannot be instantiated itself because it
+ * needs a custom implementation of a level.
+ */
+@Getter
+@AllArgsConstructor
+@SuperBuilder(toBuilder = true)
+public abstract class FRBR {
+  private final Node node;
+  private final String normEli;
+  private final String nodeName;
+
+  /**
+   * Returns the Eli as {@link String} from the FRBRThis of the specific FRBR level.
+   *
+   * @return An Eli
+   */
+  public String getEli() {
+    final String xpath = "./FRBRthis/@value";
+    return NodeParser.getValueFromExpression(xpath, node)
+        .orElseThrow(() -> new MandatoryNodeNotFound(xpath, this.nodeName, this.normEli));
```

Comment:
i'm a bit unsure how nice it is to carry these information around everywhere just for the error messages. It might be possible to get them from `node.getOwnerDocument()`?

Reply:
This was something I thought about a lot... and I was about to move the getEli to the FRBRExpression, since it would logically belong there but the ELI is something we use everywhere and it is nice to just do norm.getEli()

About passing it around this was my best approach, since the exception would me meaningless if we don't know on which law (eli) the error happened.

But to get it from the owner document would mean you need to pass the Document to a Norm again, so that you can call the getEli() method... don't really think this would be a good approach.

Another idea would be to set the eli on a process (API call, service, bla bla) level so that you can always access it, doesn't matter where. But this would require much more work specially because we don't always use the ELI from the API endpoint but rather from a norm we loaded from the db

If we don't have a better approach now, I would say let's pass the eli around for now. What do you say?

Reply:
Yeah the information is really helpful to have here! 

I also like your idea of keeping it as information on the process but that really seems like a lot of work and thought that we would need to invest into that.

But I would actually prefer to calculate the eli from the document again instead of passing it around (we need to keep infinite loops in mind here), as it would keep the api simpler and we can encapsulate all this logic to one place (otherwise we would need to add these fields to all of these classes where we want to have mandatory nodes). Maybe it would actually be a better idea to use the xpath for getting the eli directly on the document, even if that duplicates some code, instead of converting the document to a norm.

Also do we need the `nodeName` or could that just be `node.getNodeName()`?

Reply:
With the nodeName jep we don't need it. Done. And with the getEli I put it close to the exception which is now only thrown in the NodeParser, with a new method for getting mandatory nodes (and values)